h2o-algos/src/main/java/hex/pca/PCA.java

```diff
@@ -314,18 +309,23 @@ public void computeImpl() {
           // Compute SVD of Gram A'A/n using JAMA library
           // Note: Singular values ordered in weakly descending order by algorithm
           _job.update(1, "Calculating SVD of Gram matrix locally");
-          Matrix gramJ = _wideDataset ? new Matrix(ogtsk._gram.getXX()) : new Matrix(gtsk._gram.getXX());
-          SingularValueDecomposition svdJ = gramJ.svd();
+          DenseMatrix gramJ = _wideDataset ? new DenseMatrix(ogtsk._gram.getXX()) : new DenseMatrix(gtsk._gram.getXX());
```

Comment:
@mmalohlava @wendycwong 
Should we create a new JIRA issue? So that we some day in future we could compare performance  sparse vs. dense vs. general matrix PCA.